### ISSUE TITLE
[1.14] Game Crashes without Logs when Opening Geolosys Field Manual

### ISSUE THREAD

**Issue by oitsjustjose** [tterrag1098/Blur#40]
Hey there! I'm the dev of Geolosys and in 1.14 (but not 1.12), I've noticed that Blur and the Geolosys Field Manual no longer play nicely (see https://github.com/oitsjustjose/Geolosys/issues/233). I know you know what you're doing 10000% more than I do, so I imagine this is an issue with the implementation of the GUI on my side. 

Any thoughts right off the top of your head?

Let me know if you need any more info that isn't in the other post. Obviously blacklisting the GUI works, but once again I imagine this is something that I'm doing that's totally stupid.

Thanks!

**Comment by tterrag1098** [comment]
What do you mean without logs? The game cannot just close without there being an exception.

**Comment by oitsjustjose** [comment]
No, it's pretty much without logs. The game crashes, and on my MacBook (because I'm away from my desktop) macOS reports that it's a critical java error, and the logs do not say anything. It's odd because I got it to work with the GUI *once*, but as soon as I re-opened it it crashed per-usual.

[Logs](https://gist.github.com/c8eaa4e8bc5794ce7dd607f429505de7)

EDIT: I'm a dumb

I just found this in the log - I haven't seen it in the past when hunting for the stack trace but this time [I've found it](https://gist.github.com/oitsjustjose/c8eaa4e8bc5794ce7dd607f429505de7#file-latest-log-L74-L96)

**Comment by ejektaflex** [comment]
I've had the same thing just occur. I made a new instance, Forge 28.2.1, with Blur & Slice of Life: Carrot Edition. Running them both will crash the game when giving yourself the SOL book with `/give username solcarrot:food_book 1` and using it. Right clicking to open this book crashes the game instantly without anything marked at the end of the log. It crashes so fast that it registers as a right click in my web browser, haha. Nothing gets added to the log related to blur, and nothing gets logged at the time of crash, so the file looks completely normal.

Anyways, I did fix it by adding `com.cazsius.solcarrot.client.gui.FoodBookScreen` to the `guiExclusions` list in your config, but I assume something is conflicting with some of these screen classes.

**Comment by tterrag1098** [comment]
Looks like an internal bug in java's reflection libraries. In the log your java version is `1.8.0_74` which is insanely outdated. Try updating.

**Comment by ejektaflex** [comment]
Pretty sure mine is AdoptOpenJDK 1.8.0_252 or some such and I still got errors, though I haven't had any log so I can't tell you much more.
